test/linter-test.js

```diff
@@ -279,13 +329,127 @@ describe('linter', () => {
 
       });
 
+      describe('--parser-options', () => {
+
+        it('fails with parse error on test/fixture/es6.txt', () => {
+          const out = linter.invoke(cwd, [
+            '--parser-options=ecmaVersion:5', fixture_es6, '-f', 'unix'
+          ]);
+
+          const space = out.indexOf(' ');
+          const slash = out.lastIndexOf('/', space);
+          const newline = out.indexOf('\n');
+
+          assert.equals(out.substring(slash, space), '/es6.txt:3:1:');
```

Comment:
Did you know that `referee` has `assert.match(actual, expectation)`? If you invoke it with a string, it does a substring match. Just saying, we can keep this the way it is :)

Reply:
Ok! I copy-paste this part=) I can change it, but then better change it everywhere...

Reply:
No worries. Leave it as it is.